internal/resource/route.go

```diff
@@ -126,13 +127,19 @@ func (r Route) ProcessDefaults(ctx context.Context) error {
 	if r.Route == nil {
 		return fmt.Errorf("invalid nil resource")
 	}
-	if len(r.Route.Protocols) == 0 {
-		err := mergo.Merge(r.Route, defaultRoute,
-			mergo.WithTransformers(wrappersPBTransformer{}))
-		if err != nil {
-			return err
+	// When protocols has 'grpc' or 'grpcs', 'strip_path' cannot be set.
+	// If 'strip_path' is null, the default value (true) would be applied,
+	// so we set it to 'false' in order to prevent that.
+	if lo.Contains(r.Route.Protocols, "grpc") || lo.Contains(r.Route.Protocols, "grpcs") {
```

Comment:
No test for this?

Reply:
It's already there: https://github.com/Kong/koko/blob/main/internal/resource/route_test.go#L901-L927

Reply:
Ah part of the validation test, that works w/ me.